RecoHGCal/TICL/plugins/TrackstersMergeProducer.cc

```diff
@@ -528,6 +528,11 @@ void TrackstersMergeProducer::produce(edm::Event &evt, const edm::EventSetup &es
     }
   }
 
+  // Compute timing
+  for (auto &cand : *resultCandidates) {
+    cand.computeTime();
```

Comment:
In general, it's a pity to call this method on all the `TICLCandidate`s. Would it make sense to add the timing information every time we call `addTrackster` on the `tmpCandidate` and trigger the update of the timing information only for the cases in which the `TICLCandidate` has more than one unique associated `trackster`?

Reply:
Done:
https://github.com/cms-sw/cmssw/pull/32046/commits/e23de8a5c411e3cc99ddda39f11555979174d8fa?file-filters%5B%5D=